packages/http-cache-storage-lru/lib/HttpCacheStorageLru.ts

```diff
@@ -0,0 +1,34 @@
+import type { IHttpCacheStorage, IHttpCacheStorageValue } from '@comunica/http-cache-storage';
+import * as LRU from 'lru-cache';
```

Comment:
@jeswr Any idea why the depcheck task might be reporting `lru-cache` as an extra dependency that should be removed?

https://github.com/comunica/comunica/runs/6961732697?check_suite_focus=true

Reply:
It seems that the [associated package.json](https://github.com/comunica/comunica/blob/95d7313f7454c7de25fad7c501394c80b4684617/packages/http-cache-storage-lru/package.json) is missing the "files" parameter which depcheck is using to determine which files to include dependencies from.

Reply:
Aha, thanks @jeswr!

@jaxoncreed It looks like some other package.json entries are missing as well. This might be a good reference: https://github.com/comunica/comunica/blob/95d7313f7454c7de25fad7c501394c80b4684617/packages/types/package.json

Reply:
Got it